### PR TITLE
Enrich examples to boost example quality KPI - Batch 1

### PR DESCRIPTION
**Pull request by SteveMutungi254**
Enrich examples to boost example quality KPI

### CODE REVIEW

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationProxyApplication.md [hunk 1]
```diff
@@ -59,9 +60,7 @@ This example retrieves an application configured for Application Proxy.
 
 ### -ApplicationId
 
-This ApplicationId is the unique application ID of the application.
-This ApplicationId can be found using the `Get-EntraBetaApplication` command.
-You can also find ApplicationId in the Microsoft Portal by navigating to Microsoft Entra ID, Enterprise Applications, All Applications, Select your application, go to the properties tab, and use the ObjectId on that page.
+The ApplicationId is a unique identifier for the application. You can find it using the Get-EntraBetaApplication command in PowerShell, or in the Microsoft Entra admin portal by navigating to Entra ID > Enterprise Applications > All Applications, selecting your application, and viewing the Properties tab. Use the `ObjectId` value.
```
Comment (msewaweru) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationProxyApplication.md:63]:
```suggestion
The ApplicationId is a unique identifier for the application. You can find it using the `Get-EntraBetaApplication` command in PowerShell, or in the Microsoft Entra admin portal by navigating to **Entra ID** > **Enterprise Applications** > **All Applications**, selecting your application, and viewing the **Properties** tab. Use the `ObjectId` value.
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraApplicationKey.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Adds a new key to an application.
 Connect-Entra -Scopes 'Application.ReadWrite.All','Application.ReadWrite.OwnedBy'
 $app = Get-EntraApplication -Filter "DisplayName eq '<application-display-name>'"
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraApplicationKey.md:47]: For conformity:
```suggestion
$application = Get-EntraApplication -Filter "DisplayName eq '<application-display-name>'"
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraApplicationKey.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Adds a new key to an application.
...
 $params = @{
-    ObjectId = $app.ObjectId
+    ObjectId = $app.Id
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraApplicationKey.md:49]: Update the parameter descriptions after this example as well.
```suggestion
    ApplicationId = $application.Id
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraApplicationPassword.md [hunk 4]
```diff
@@ -42,19 +42,14 @@ Adds a strong password to an application.
...
-
-New-EntraApplicationPassword @params
+$application = Get-EntraApplication -Filter "DisplayName eq 'Contoso Helpdesk Application'"
+$passwordCredential= New-Object Microsoft.Open.MSGraph.Model.PasswordCredential
+$passwordCredential.StartDateTime = Get-Date -Year 2024 -Month 12 -Day 28
+$passwordCredential.EndDateTime = Get-Date -Year 2025 -Month 2 -Day 28
+$passwordCredential.KeyId = 'bbbbbbbb-1c1c-2d2d-3e3e-444444444444'
+$passwordCredential.CustomKeyIdentifier = [System.Text.Encoding]::UTF8.GetBytes('a')
+$passwordCredential.Hint = 'b'
+New-EntraApplicationPassword -ObjectId $application.Id -PasswordCredential $passwordCredential
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/New-EntraApplicationPassword.md:52]:
```suggestion
New-EntraApplicationPassword -ApplicationId $application.Id -PasswordCredential $passwordCredential
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ Other services can use the information stored in the ServiceEndpoint entity to f
 ```powershell
 Connect-Entra -Scopes 'Application.Read.All'
 $application = Get-EntraApplication -Filter "DisplayName eq 'Contoso Helpdesk Application'"
-Get-EntraApplicationServiceEndpoint -ApplicationId $application.ObjectId
+Get-EntraApplicationServiceEndpoint -ApplicationId $application.Id
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md:51]: Fails with an error: `A parameter cannot be found that matches parameter name 'ApplicationId'.`
```suggestion
Get-EntraApplicationServiceEndpoint -ObjectId $application.Id
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md [hunk 6]
```diff
@@ -60,7 +60,7 @@ This example demonstrates how to retrieve service endpoint of the application th
 ```powershell
 Connect-Entra -Scopes 'Application.Read.All'
 $application = Get-EntraApplication -Filter "DisplayName eq 'Contoso Helpdesk Application'"
-Get-EntraApplicationServiceEndpoint -ApplicationId $application.ObjectId -All 
+Get-EntraApplicationServiceEndpoint -ApplicationId $application.Id -All
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md:63]: Same error as above.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md [hunk 7]
```diff
@@ -72,7 +72,7 @@ This example demonstrates how to retrieve all service endpoints of a specified a
 ```powershell
 Connect-Entra -Scopes 'Application.Read.All'
 $application = Get-EntraApplication -Filter "DisplayName eq 'Contoso Helpdesk Application'"
-Get-EntraApplicationServiceEndpoint -ApplicationId $application.ObjectId -Top 5
+Get-EntraApplicationServiceEndpoint -ApplicationId $application.Id -Top 5
```
Comment (msewaweru) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md:75]: Same error as above.

**Review by msewaweru — CHANGES_REQUESTED** [review]
@SteveMutungi254 - left some comments for you. Thanks.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md (hunk 6 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md:63]: Great catch!

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md (hunk 7 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationServiceEndpoint.md]: Great catch, @msewaweru!

**Review by msewaweru — APPROVED** [review]
Great job, @SteveMutungi254. Thanks.

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**